packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -445,6 +499,30 @@ class DirectStripeRoutes {
     return response;
   }
 
+  /**
+   * Gather all capabilities granted by a product across all clients
+   *
+   * @param {*} productId
+   */
+  async getProductCapabilities(productId) {
```

Comment:
This is another bit of code that probably should have been added along with direct Stripe routes.